calculate_number_of_profiled_coding_base_pairs/add_number_of_profiled_coding_base_pairs.py

```diff
@@ -0,0 +1,94 @@
+import csv
+import pandas as pd
+import os
+import sys
+import argparse
+import ntpath
+
+def generate_dict(key, value, dictionary):
+    if not pd.isnull(key):
+        if ';' not in key:
+            dictionary[key] = value
+        else:
+            genes = key.split('; ')
+            for gene in genes:
+                dictionary[gene] = value
+
+def update_file(args):
+    inputFile = args.inputFile
+    uniprotFile = args.uniprotFile or os.getcwd() + "/uniprot_gene_name_with_protein_length.tab"
```

Comment:
For some reason `os.getcwd()` here doesn't seem to work when I run the script. Therefore I always have to define the -u and add the absolute path, to have the script be able to find the uniprot tab file.

Reply:
@yichaoS That's weird, I test it locally without `-u` and it works fine. Could you send me the error message?

Reply:
@leexgh I was running the script under a different location than the script folder itself. So the os.getcwd() was reading the wrong path. Please ignore this.